bodhi/server/util.py

```diff
@@ -35,11 +35,15 @@
 import arrow
 import bleach
 import colander
+import createrepo_c as cr
+import hawkey
+import libcomps
 import libravatar
 import librepo
 import markdown
 import requests
 import rpm
+import shutil
```

Comment:
For PEP-8, this import should be included with the first group of imports, since it's part of the Python stdlib.